### PR TITLE
[OSDOCS-11353] Document the whitelist IPs required for the ingress access to the API server

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.16+
Issue:

https://issues.redhat.com/browse/OSDOCS-11353
Link to docs preview:

https://79139--ocpdocs-pr.netlify.app/openshift-dedicated/latest/security/rh-required-whitelisted-ip-addresses-for-sre-access
https://79139--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_planning/gcp-ccs.html#ccs-gcp-requirements-security_gcp-ccs

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jaybeeunix** [comment]
from a technical point-of-view, it looks good to me. i mentioned to @mletalie that we aren't consistent with "whitelist" vs. "allowlist", but that's my only nit.

**Comment by mletalie** [comment]
Hello @xueli181114, May I get a review for this PR when you get a moment? Thanks!

File: modules/ccs-gcp-customer-requirements.adoc [hunk 1]
```diff
@@ -62,6 +62,11 @@ This policy only provides Red Hat with permissions and capabilities to change re
 
 * Volume snapshots will remain within the customer-provided GCP account and customer-specified region.
 
-* Red Hat must have ingress access to the API server through white-listed Red Hat machines.
-
+* Red Hat must have ingress access to the API server through allowlist IP addresses.
++
+[NOTE]
+====
+For information regarding allowlist IP addresses, see Additional resources.
```
Comment (lpettyjo) [inline modules/ccs-gcp-customer-requirements.adoc:69]: Rewrite as: "For information about allowlist..."

File: security/rh-required-whitelisted-IP-addresses-for-sre-access.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+= Required allowlist IP addresses for SRE cluster access
+
+:context: rh-required-whitelisted-IP-addresses-for-sre-access
+
+toc::[]
+
+[id='required-whitelisted-overview']
+== Overview
+
+In order for Red Hat SREs to troubleshoot any issues within {product-title} clusters, they must have ingress access to the API server through allowlist IP addresses.
```
Comment (lpettyjo) [inline security/rh-required-whitelisted-IP-addresses-for-sre-access.adoc:14]: Rewrite as: "For Red Hat SREs to troubleshoot any issues within..."

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM!

File: security/rh-required-whitelisted-IP-addresses-for-sre-access.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+[id="rh-required-whitelisted-IP-addresses-for-sre-access_{context}"]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+include::_attributes/common-attributes.adoc[]
+= Required allowlist IP addresses for SRE cluster access
+
+:context: rh-required-whitelisted-IP-addresses-for-sre-access
+
+toc::[]
+
+[id='required-whitelisted-overview']
```
Comment (jldohmann) [inline security/rh-required-whitelisted-IP-addresses-for-sre-access.adoc:11]:
```suggestion
[id="required-whitelisted-overview_{context}"]
```

File: security/rh-required-whitelisted-IP-addresses-for-sre-access.adoc [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+== Overview
+
+For Red Hat SREs to troubleshoot any issues within {product-title} clusters, they must have ingress access to the API server through allowlist IP addresses.
+
+[id='required-whitelisted-access']
```
Comment (jldohmann) [inline security/rh-required-whitelisted-IP-addresses-for-sre-access.adoc:16]:
```suggestion
[id="required-whitelisted-access_{context}"]
```

**Review by jldohmann — COMMENTED** [review]
just a couple things before merge

**Comment by jldohmann** [comment]
feel free to ping me on slack to merge when it's ready 👍

**Comment by mletalie** [comment]
> feel free to ping me on slack to merge when it's ready 👍

All good, thanks! 
Pinged you via Slack.

**Pull request merged**